src/queries/ProposalBookingQueries.ts

```diff
@@ -35,4 +36,24 @@ export default class ProposalBookingQueries {
 
     return proposalBooking;
   }
+
+  @Authorized()
+  async getByProposalId(
+    ctx: ResolverContext,
+    proposalId: number,
+    filter?: ProposalProposalBookingFilter
+  ): Promise<ProposalBooking | null> {
+    const proposalBooking = await this.proposalBookingDataSource.getByProposalId(
+      proposalId,
+      filter
+    );
+
+    if (!proposalBooking) {
+      return null;
+    }
+
+    // await helperInstrumentScientistHasAccess(ctx, proposalBooking);
```

Comment:
Do we need this commented part here or?

Reply:
Yes, it's related to the permission thing I mentioned.